test/torchaudio_unittest/example/hubert/test_crop_audio_label.py

```diff
@@ -0,0 +1,46 @@
+import math
+
+import torch
+from dataset.hubert_dataset import _crop_audio_label
+from parameterized import parameterized
+from torchaudio_unittest.common_utils import get_whitenoise, TorchaudioTestCase
+
+
+class TestCropAudioLabel(TorchaudioTestCase):
+    @classmethod
+    def setUpClass(cls) -> None:
+        super().setUpClass()
+        torch.random.manual_seed(31)
+
+    @parameterized.expand(
+        [
+            (400,),
+            (800,),
+        ]
+    )
+    def test_zero_offset(self, num_frames):
```

Comment:
Could you add description of what "you" intend to test here, so that future maintainers without context can understand it easily.

In general, bugs in tests are harder to detect, so having description of "(under condition C) doing A should produce B", help a lot. Without such description, future maintainers have to 1. first figure out what test does, 2. try to make sense of it, and 3. debug. Which is very difficult.